src/components/App/index.js

```diff
@@ -43,40 +48,53 @@ const App = () => {
     setExportHref(getExportHrefValue(playList))
   }, [playList])
 
+  useEffect(() => {
+    const root = window.document.documentElement
+    root.classList.remove(colorTheme === 'dark' ? 'light' : 'dark')
+    root.classList.add(colorTheme)
+    localStorage.setItem('theme', colorTheme)
+  }, [colorTheme])
+
   return html`
-    <div class="p-4">
+    <div class="p-4 dark:bg-[#18181b] min-h-screen transition duration-500">
       <div class="flex align-center mb-4">
         <img class="w-10 sm:w-20 mr-3 object-contain" src=${svgLogo} alt="logo" />
-        <h1 class="font-medium leading-tight text-2xl sm:text-4xl lg:text-5xl leading-none">Anonymous youtube playlist generator</h1>
+        <h1 class="font-medium leading-tight text-2xl sm:text-4xl lg:text-5xl leading-none dark:text-slate-300">Anonymous youtube playlist generator</h1>
       </div>
-      <p class="mb-4">Create youtube playlists without a google account</p>
+      <p class="mb-4 dark:text-slate-300">Create youtube playlists without a google account</p>
       <hr class="mb-4"/>
+
+      <div class="fixed bottom-0 right-0 p-5">
+        <span onClick=${() => setColorTheme(oldColorTheme => oldColorTheme === 'dark' ? 'light' : 'dark')} class="w-10 h-10 bg-primary-color block rounded-full shadow-lg cursor-pointer text-white flex items-center justify-center transition duration-500">
+          <${colorTheme === 'light' ? light : dark} customClass="h-6 pr-[0.2rem]" width="full" fill="white" />
```

Comment:
I think this is the last thing to fix. This is weird, but it looks like that this logic is inverted. Basically when I'm in the night mode, it shows the moon instead of sun and vice versa
![image](https://user-images.githubusercontent.com/30677819/171689007-0f1bf6ee-f805-46ba-892c-82b9e799d7ba.png)
![image](https://user-images.githubusercontent.com/30677819/171689037-447985bd-e418-4019-9eba-4e49ba6f8a0f.png)

Reply:
I'll change it but this was your request....

> When in "dark mode" the icon should be the [moon icon](https://fontawesome.com/search?q=moon&s=solid%2Cbrands)
> When in "light mode" the icon should be the [sun icon](https://fontawesome.com/search?q=sun&s=solid%2Cbrands)

Reply:
HAHAHA You are right! I messed up